Source/ExecutionEngine/ExecutionEngine.cs

```diff
@@ -1244,6 +1244,13 @@ private static void CleanupCheckers(string requestId)
             verificationResult.Errors = null;
             verificationResult.Outcome = VCGen.Outcome.Inconclusive;
           }
+          catch(AggregateException e)
+          {
+            printer.AdvisoryWriteLine("Advisory: {0} SKIPPED because: {1}",
+              impl.Name, e.Message);
+            verificationResult.Errors = null;
+            verificationResult.Outcome = VCGen.Outcome.Inconclusive;
```

Comment:
Inconclusive means that the solver ran but couldn't prove unsat or sat right? For the scenario of invalid solver arguments I wouldn't mind an additional enum value, like `InvalidConfiguration`.